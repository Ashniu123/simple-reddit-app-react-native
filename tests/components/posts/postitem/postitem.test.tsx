import * as React from 'react';
import { shallow } from 'enzyme';

import PostItem from '../../../../src/components/posts/postitem/postitem.component';

test('should render PostItem component correctly', () => {
	const wrapper = shallow(<PostItem />);
	expect(wrapper).toMatchSnapshot();
});