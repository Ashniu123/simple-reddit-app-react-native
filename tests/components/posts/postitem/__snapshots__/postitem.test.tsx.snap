// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PostItem component correctly 1`] = `
<Component
  style={
    Object {
      "borderTopColor": "rgba(0,0,0,0.3)",
      "borderTopWidth": 1,
      "flexDirection": "row",
      "height": 100,
      "padding": 15,
    }
  }
>
  <Component
    style={
      Object {
        "width": 80,
      }
    }
  >
    <Component
      style={
        Object {
          "backgroundColor": "#eee",
          "height": 60,
          "width": 60,
        }
      }
    />
  </Component>
  <Component
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      numberOfLines={2}
      style={Object {}}
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#999",
          "fontSize": 11,
          "fontWeight": "bold",
        }
      }
    />
  </Component>
</Component>
`;
